modules/tensor_mechanics/test/tests/beam/action/tests

```diff
@@ -1,109 +1,174 @@
     expect_err = 'LineElementAction: `nodal_Ixx`, `nodal_Iyy`, `nodal_Izz` should be provided as input to calculate inertial torque on beam due to nodal inertia.'
+    requirement = "The LineElementAction class shall produce an error if nodal inertia"
+                  " is not provided as input for dynamic simulations using the nodal"
+                  " inertia matrix."
   [../]
   [./beam_action_test14]
     type = 'RunException'
     input = 'beam_action_chk.i'
     cli_args = 'Modules/TensorMechanics/LineElementMaster/block_1/displacements="disp_x" Modules/TensorMechanics/LineElementMaster/block_1/rotations="rot_x" Modules/TensorMechanics/LineElementMaster/block_1/area=1.0 Modules/TensorMechanics/LineElementMaster/block_1/rotational_velocities="rot_vel_x" Modules/TensorMechanics/LineElementMaster/block_1/beta=0.25 Modules/TensorMechanics/LineElementMaster/block_1/density=1.0 Modules/TensorMechanics/LineElementMaster/block_1/nodal_mass=0.1 Modules/TensorMechanics/LineElementMaster/block_1/nodal_Ixx=0.1 Modules/TensorMechanics/LineElementMaster/block_all/block="1 2"'
     expect_err = 'LineElementAction: The block restrictions in the LineElement actions must be non-overlapping.'
+
+    requirement = "The LineElementAction class shall produce an error if multiple subblocks"
+                  " specify properties for the same mesh block."
   [../]
   [./beam_action_test15]
     type = 'RunException'
     input = 'beam_action_chk.i'
     cli_args = 'Modules/TensorMechanics/LineElementMaster/block_1/displacements="disp_x" Modules/TensorMechanics/LineElementMaster/block_1/rotations="rot_x" Modules/TensorMechanics/LineElementMaster/block_1/area=1.0 Modules/TensorMechanics/LineElementMaster/block_1/rotational_velocities="rot_vel_x" Modules/TensorMechanics/LineElementMaster/block_1/beta=0.25 Modules/TensorMechanics/LineElementMaster/block_1/density=1.0 Modules/TensorMechanics/LineElementMaster/block_1/nodal_mass=0.1 Modules/TensorMechanics/LineElementMaster/block_1/nodal_Ixx=0.1'
     expect_err = 'LineElementAction: No LineElement action can be block unrestricted if more than one LineElement action is specified.'
+
+    requirement = "The LineElementAction class shall produce an error if an action subblock"
+                  " is mesh block restricted while another is not."
   [../]
   [./beam_action_test16]
     type = 'RunException'
     input = '2_block_common.i'
     cli_args = 'Modules/TensorMechanics/LineElementMaster/dynamic_nodal_translational_inertia=true'
     expect_err = 'LineElementAction: Variable names for translational and rotational velocities and accelerations should be provided as input to perform dynamic simulation using beam elements'
+
+    requirement = "The LineElementAction class shall produce an error if dynamic simulation"
```

Comment:
This makes it sound like there's a 'dynamic simulation' input parameter, but there isn't one.